src/js/hca/containers/ConfirmationPage.jsx

```diff
@@ -22,33 +21,38 @@ export class ConfirmationPage extends React.Component {
   }
 
   render() {
-    const time = _.get('form.submission.response.timestamp', this.props);
+    const { submission, data } = this.props.form;
+    const { response } = submission;
+    const name = data.veteranFullName;
 
     return (
       <div>
-        <div className="usa-width-one-sixth medium-2 columns">
-          <i className="fa fa-check-circle hca-success-icon"></i>
-        </div>
-        <div className="usa-width-five-sixths medium-10 columns">
-          <h4 className="success-copy">You have successfully submitted your application for health care.</h4>
+        <h3 className="confirmation-page-title">Claim received</h3>
+        <p>Normally processed within <strong>a week</strong>.</p>
```

Comment:
We should make sure this is ok to put on here. Where did this timeframe come from?

Reply:
The timeframe is in the design, but the content was [updated](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2965#issuecomment-332610175) by @peggygannon, and has been changed to reflect her note.

Reply:
@stacyhlinn @peggygannon is the timeframe here confirmed?